In-tree build errors with `emconfigure: command not found` during `make -C cpython` while following initial setup instructions

## 🐛 Bug

Following the [in-tree build instructions](https://pyodide.org/en/stable/development/new-packages.html#prerequisites) gives this error:

```
...
patching file Lib/importlib/_bootstrap.py
patching file Python/importlib.h
patching file Lib/ctypes/util.py
patching file Lib/multiprocessing/connection.py
touch /src/cpython/build/Python-3.10.2/.patched
# --enable-big-digits=30 :
#   Python integers have "digits" of size 15 by default on systems with 32
#   bit pointers and size 30 on systems with 16 bit pointers. Python uses
#   "digits" of size 15 by default on systems with 32 bit pointers and size
#   30 on systems with 16 bit pointers. WASM has 32 bit pointers so Python
#   will default to the size 15 digits but WASM has native 64 bit arithmetic
#   so it is more efficient to use 30 bit digits.
cp config.site /src/cpython/build/Python-3.10.2/
( \
	cd /src/cpython/build/Python-3.10.2; \
	CONFIG_SITE=./config.site READELF=true emconfigure \
	  ./configure \
		  CFLAGS="-O2 -g0 -fPIC " \
		  CPPFLAGS="-sUSE_BZIP2=1 -sUSE_ZLIB=1" \
		  PLATFORM_TRIPLET="wasm32-emscripten" \
		  --without-pymalloc \
		  --disable-shared \
		  --disable-ipv6 \
		  --enable-big-digits=30 \
		  --enable-optimizations \
		  --host=wasm32-unknown-emscripten\
		  --build=x86_64-pc-linux-gnu \
		  --prefix=/src/cpython/installs/python-3.10.2  \
)
/bin/bash: line 2: emconfigure: command not found
make: *** [Makefile:80: /src/cpython/build/Python-3.10.2/Makefile] Error 127
make: Leaving directory '/src/cpython'
```

### To Reproduce

I followed the instructions described here: https://pyodide.org/en/stable/development/new-packages.html#prerequisites
```
git clone https://github.com/pyodide/pyodide
cd pyodide
./run_docker
pip install -e pyodide-build
make -C cpython
```

<details>
<summary>Full logs</summary>

```
joe@pop-os:~/pyodide$ git clone https://github.com/pyodide/pyodide
Cloning into 'pyodide'...
remote: Enumerating objects: 23970, done.
remote: Counting objects: 100% (124/124), done.
remote: Compressing objects: 100% (101/101), done.
remote: Total 23970 (delta 37), reused 85 (delta 21), pack-reused 23846
Receiving objects: 100% (23970/23970), 21.64 MiB | 16.00 MiB/s, done.
Resolving deltas: 100% (15949/15949), done.
joe@pop-os:~/pyodide$ cd pyodide
joe@pop-os:~/pyodide/pyodide$ ./run_docker
Unable to find image 'pyodide/pyodide-env:20221102-chrome107-firefox106' locally
20221102-chrome107-firefox106: Pulling from pyodide/pyodide-env
15115158dd02: Already exists
4d445d10bda3: Already exists
cc7ba5177a87: Already exists
a4759db5e555: Already exists
57e81f2987b5: Already exists
e27ec2bfa589: Pulling fs layer
bc88138a0687: Pulling fs layer
20709cdee99b: Pulling fs layer
6c41703e34d6: Pulling fs layer
ff817c04448f: Pulling fs layer
07eb47de68f5: Pulling fs layer
656c70b65803: Pulling fs layer
82469f70fc31: Pulling fs layer
e17fb27f4afa: Pulling fs layer
3a7fc13009c6: Pulling fs layer
aa8513f8a41e: Pulling fs layer
07eb47de68f5: Waiting
6c41703e34d6: Waiting
ff817c04448f: Waiting
e17fb27f4afa: Waiting
3a7fc13009c6: Waiting
656c70b65803: Waiting
bc88138a0687: Verifying Checksum
bc88138a0687: Download complete
20709cdee99b: Verifying Checksum
20709cdee99b: Download complete
6c41703e34d6: Verifying Checksum
6c41703e34d6: Download complete
07eb47de68f5: Verifying Checksum
07eb47de68f5: Download complete
656c70b65803: Verifying Checksum
656c70b65803: Download complete
ff817c04448f: Verifying Checksum
ff817c04448f: Download complete
82469f70fc31: Verifying Checksum
82469f70fc31: Download complete
e17fb27f4afa: Verifying Checksum
e17fb27f4afa: Download complete
aa8513f8a41e: Verifying Checksum
aa8513f8a41e: Download complete
e27ec2bfa589: Download complete
3a7fc13009c6: Verifying Checksum
3a7fc13009c6: Download complete
e27ec2bfa589: Pull complete
bc88138a0687: Pull complete
20709cdee99b: Pull complete
6c41703e34d6: Pull complete
ff817c04448f: Pull complete
07eb47de68f5: Pull complete
656c70b65803: Pull complete
82469f70fc31: Pull complete
e17fb27f4afa: Pull complete
3a7fc13009c6: Pull complete
aa8513f8a41e: Pull complete
Digest: sha256:384f7b3325b8cb152e56ce59fc23f2146435141c1307b4c0d10cdfd9f2829c55
Status: Downloaded newer image for pyodide/pyodide-env:20221102-chrome107-firefox106
Waiting for initialization to finish...
joe@874e5b4763d3:/src$ pip install -e pyodide-build
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///src/pyodide-build
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: virtualenv in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (20.16.6)
Collecting types-requests
  Downloading types_requests-2.28.11.7-py3-none-any.whl (14 kB)
Requirement already satisfied: tomli in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (2.0.1)
Requirement already satisfied: wheel in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (0.37.1)
Collecting cython<3.0
  Downloading Cython-0.29.33-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.manylinux_2_24_x86_64.whl (1.9 MB)
     |████████████████████████████████| 1.9 MB 22.2 MB/s 
Collecting resolvelib
  Downloading resolvelib-0.9.0-py2.py3-none-any.whl (16 kB)
Requirement already satisfied: pyyaml in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (6.0)
Requirement already satisfied: cmake in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (3.24.2)
Requirement already satisfied: build==0.7.0 in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (0.7.0)
Collecting auditwheel-emscripten~=0.0.9
  Downloading auditwheel_emscripten-0.0.9-py3-none-any.whl (19 kB)
Collecting pyodide-cli~=0.2.1
  Downloading pyodide_cli-0.2.2-py3-none-any.whl (10 kB)
Requirement already satisfied: requests in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (2.28.1)
Requirement already satisfied: pydantic>=1.10.2 in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (1.10.2)
Collecting unearth~=0.6
  Downloading unearth-0.7.0-py3-none-any.whl (39 kB)
Collecting typer
  Downloading typer-0.7.0-py3-none-any.whl (38 kB)
Requirement already satisfied: ruamel.yaml in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (0.17.21)
Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from pyodide-build==0.22.0) (21.3)
Requirement already satisfied: pep517>=0.9.1 in /usr/local/lib/python3.10/site-packages (from build==0.7.0->pyodide-build==0.22.0) (0.13.0)
Collecting leb128
  Downloading leb128-1.0.4.tar.gz (2.6 kB)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /usr/local/lib/python3.10/site-packages (from packaging->pyodide-build==0.22.0) (3.0.9)
Requirement already satisfied: typing-extensions>=4.1.0 in /usr/local/lib/python3.10/site-packages (from pydantic>=1.10.2->pyodide-build==0.22.0) (4.4.0)
Collecting rich
  Downloading rich-13.0.1-py3-none-any.whl (238 kB)
     |████████████████████████████████| 238 kB 67.7 MB/s 
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/site-packages (from requests->pyodide-build==0.22.0) (1.26.12)
Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/site-packages (from requests->pyodide-build==0.22.0) (3.4)
Requirement already satisfied: charset-normalizer<3,>=2 in /usr/local/lib/python3.10/site-packages (from requests->pyodide-build==0.22.0) (2.1.1)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/site-packages (from requests->pyodide-build==0.22.0) (2022.9.24)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /usr/local/lib/python3.10/site-packages (from rich->pyodide-cli~=0.2.1->pyodide-build==0.22.0) (2.13.0)
Collecting commonmark<0.10.0,>=0.9.0
  Downloading commonmark-0.9.1-py2.py3-none-any.whl (51 kB)
     |████████████████████████████████| 51 kB 13.8 MB/s 
Requirement already satisfied: ruamel.yaml.clib>=0.2.6 in /usr/local/lib/python3.10/site-packages (from ruamel.yaml->pyodide-build==0.22.0) (0.2.7)
Collecting click<9.0.0,>=7.1.1
  Downloading click-8.1.3-py3-none-any.whl (96 kB)
     |████████████████████████████████| 96 kB 12.0 MB/s 
Collecting colorama<0.5.0,>=0.4.3
  Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Collecting rich
  Downloading rich-12.6.0-py3-none-any.whl (237 kB)
     |████████████████████████████████| 237 kB 89.7 MB/s 
Collecting shellingham<2.0.0,>=1.3.0
  Downloading shellingham-1.5.0.post1-py2.py3-none-any.whl (9.4 kB)
Collecting types-urllib3<1.27
  Downloading types_urllib3-1.26.25.4-py3-none-any.whl (14 kB)
Requirement already satisfied: distlib<1,>=0.3.6 in /usr/local/lib/python3.10/site-packages (from virtualenv->pyodide-build==0.22.0) (0.3.6)
Requirement already satisfied: filelock<4,>=3.4.1 in /usr/local/lib/python3.10/site-packages (from virtualenv->pyodide-build==0.22.0) (3.8.0)
Requirement already satisfied: platformdirs<3,>=2.4 in /usr/local/lib/python3.10/site-packages (from virtualenv->pyodide-build==0.22.0) (2.5.2)
Building wheels for collected packages: leb128
  Building wheel for leb128 (setup.py) ... done
  Created wheel for leb128: filename=leb128-1.0.4-py3-none-any.whl size=2461 sha256=3f58b1a456c279fa6661007ffa734ee91b974d26f973b9f51e9fb257e29a854a
  Stored in directory: /src/.docker_home/.cache/pip/wheels/15/8a/b8/0fcb902ae7a5020df8a0466773646254a356b487fcd8314a6e
Successfully built leb128
Installing collected packages: commonmark, click, typer, shellingham, rich, colorama, types-urllib3, pyodide-cli, leb128, unearth, types-requests, resolvelib, cython, auditwheel-emscripten, pyodide-build
  Running setup.py develop for pyodide-build
Successfully installed auditwheel-emscripten-0.0.9 click-8.1.3 colorama-0.4.6 commonmark-0.9.1 cython-0.29.33 leb128-1.0.4 pyodide-build pyodide-cli-0.2.2 resolvelib-0.9.0 rich-12.6.0 shellingham-1.5.0.post1 typer-0.7.0 types-requests-2.28.11.7 types-urllib3-1.26.25.4 unearth-0.7.0
WARNING: You are using pip version 21.2.4; however, version 22.3.1 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
joe@874e5b4763d3:/src$ make -C cpython
make: Entering directory '/src/cpython'
[ -d /src/cpython/downloads ] || mkdir /src/cpython/downloads
wget -q -O /src/cpython/downloads/Python-3.10.2.tgz https://www.python.org/ftp/python/3.10.2/Python-3.10.2.tgz
shasum --algorithm 256 --check checksums --quiet || (rm /src/cpython/downloads/Python-3.10.2.tgz; false)
[ -d /src/cpython/build/Python-3.10.2 ] || (mkdir -p /src/cpython/build/; tar -C /src/cpython/build/ -xf /src/cpython/downloads/Python-3.10.2.tgz)
cat patches/*.patch | (cd /src/cpython/build/Python-3.10.2 ; patch -p1)
patching file Objects/call.c
patching file Objects/descrobject.c
patching file Objects/methodobject.c
patching file config.sub
patching file configure
patching file configure.ac
patching file Modules/posixmodule.c
patching file Lib/platform.py
patching file Lib/test/libregrtest/main.py
patching file Lib/test/support/script_helper.py
patching file Lib/test/test_cgi.py
patching file Lib/test/test_fcntl.py
patching file Lib/test/test_gzip.py
patching file Lib/test/test_itertools.py
patching file Modules/Setup
patching file Modules/_ctypes/cfield.c
patching file Lib/test/test_code.py
patching file Python/importdl.c
patching file Objects/methodobject.c
patching file Misc/NEWS.d/next/Library/2022-03-08-11-34-06.bpo-23325.3VQnfo.rst
patching file Modules/signalmodule.c
patching file Modules/_ctypes/callbacks.c
patching file Modules/_ctypes/callproc.c
patching file Modules/signalmodule.c
patching file Python/ceval.c
patching file Modules/main.c
patching file Lib/importlib/_bootstrap.py
patching file Python/importlib.h
patching file Lib/ctypes/util.py
patching file Lib/multiprocessing/connection.py
touch /src/cpython/build/Python-3.10.2/.patched
# --enable-big-digits=30 :
#   Python integers have "digits" of size 15 by default on systems with 32
#   bit pointers and size 30 on systems with 16 bit pointers. Python uses
#   "digits" of size 15 by default on systems with 32 bit pointers and size
#   30 on systems with 16 bit pointers. WASM has 32 bit pointers so Python
#   will default to the size 15 digits but WASM has native 64 bit arithmetic
#   so it is more efficient to use 30 bit digits.
cp config.site /src/cpython/build/Python-3.10.2/
( \
	cd /src/cpython/build/Python-3.10.2; \
	CONFIG_SITE=./config.site READELF=true emconfigure \
	  ./configure \
		  CFLAGS="-O2 -g0 -fPIC " \
		  CPPFLAGS="-sUSE_BZIP2=1 -sUSE_ZLIB=1" \
		  PLATFORM_TRIPLET="wasm32-emscripten" \
		  --without-pymalloc \
		  --disable-shared \
		  --disable-ipv6 \
		  --enable-big-digits=30 \
		  --enable-optimizations \
		  --host=wasm32-unknown-emscripten\
		  --build=x86_64-pc-linux-gnu \
		  --prefix=/src/cpython/installs/python-3.10.2  \
)
/bin/bash: line 2: emconfigure: command not found
make: *** [Makefile:80: /src/cpython/build/Python-3.10.2/Makefile] Error 127
make: Leaving directory '/src/cpython'
```
</details>

--- a/docs/development/new-packages.md
+++ b/docs/development/new-packages.md
@@ -83,6 +83,7 @@ If you want to build the package, you will need to build Python which you can do
 as follows:
 
 ```bash
+make -C emsdk
 make -C cpython
 ```
 

